Src/Test.UnitTests.Sarif.PatternMatcher.Cli/AnalyzeCommandTests.cs

```diff
@@ -17,6 +19,26 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Cli
 {
     public class AnalyzeCommandTests
     {
+        [Fact]
+        public void AnalyzeCommand_DefinitionsArgumentIsRequired()
```

Comment:
>public void AnalyzeCommand_DefinitionsArgumentIsRequired() [](http://example.com/codeflow?start=8&length=58)

This test isn't currently passing via the PR build pipeline or locally (using BuildAndTest.cmd).